content/tracing/services/trace.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Trace
+kind: Documentation
+further_reading:
+- link: "/tracing/services/service"
+  text: Learn more about services in Datadog
+- link: "/tracing/services/resource"
+  text: "Dive into your resource performances and traces"
+---
+
+<div class="alert alert-info"> 
+<a href="https://docs.datadoghq.com/getting_started/apm_tracing">Read our APM getting started page</a>to have a bird-eyed view of all tracing concepts
+</div>
+
+## Definition
```

Comment:
Same as on other pages.  I think this page can just focus on what information you're getting on our traces pages.  Right now I find this confusing.  I think all we need to cover is:

* APM lets you visualize traces in 2 ways
  * *Flame Graph* - this is chronological
    * Click any span to see metadata
        * If you don't see the metadata you want OOTB - you can instrument your code to send over more info.  See our docs for: [Java](https://docs.datadoghq.com/tracing/languages/java/#manual-instrumentation), Ruby, Python, Go.
    * Click on host tab to see info about the host
    * Color spans by service (same color you set on the services list page) or host
  * *List View* - Same information sorted by service or host
    * Choose what metics you want to see and sort by